AssetManagementSystem/Utility/Logger.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using AssetManagementSystem.Models;
+using Utility;
+
+namespace AssetManagementSystem.Utility
+{
+    public static class Logger
+    {
+        private static AssetManagementSystemContext db = new AssetManagementSystemContext();
+
+        //
+        // returns 0 on failure, else returns id of new Record in Database
+        //
+        // pass the exception and Level of the log
+        // the function will pull out the details from exception
```

Comment:
Anytime a log record is written, a message should be provided. This is distinct from an exception message which tends to be programmer related.

It would be better to have a single `Record` method that takes a string message, a log level value, and optionally an exception.

Reply:
Yes I think I left a few testing relics in there.   I'll make sure to clean it up  on the next branch.